sky/provision/kubernetes/utils.py

```diff
@@ -45,6 +45,13 @@
     jinja2 = adaptors_common.LazyImport('jinja2')
     yaml = adaptors_common.LazyImport('yaml')
 
+# Please be careful when changing this.
+# When mounting, Kubernetes changes the ownership of the parent directory
+# to root:root.
+# See https://stackoverflow.com/questions/50818029/mounted-folder-created-as-root-instead-of-current-user-in-docker/50820023#50820023.  # pylint: disable=line-too-long
+HIGH_AVAILABILITY_DEPLOYMENT_VOLUME_MOUNT_NAME = 'sky-data'
+HIGH_AVAILABILITY_DEPLOYMENT_VOLUME_MOUNT_PATH = '/home/sky'
```

Comment:
How do we enforce the user in controller container? I think the home dir varies for different images used

Reply:
Yes but for kubernetes I think it's quite the same.

Reply:
Maybe we can leave some comments here?

Reply:
Exactly. Let's directly use `~`.

Reply:
Well, yes. So we need another `docker ... whoami` like in `docker_utils`? Since the k8s mountPath requires a absolute path.